dod-advana/gamechanger-web
# Feature/UOT-109813 JAIC AI Clone

## Description

- Create initial version of JAIC AI clone
- all the needed clone modules created
- clone home page includes basic search and loads data from ES (no filter functionality)
- JAIC AI profile page UI (hardcoded, no populated data, search does nothing) built out on separate route: /budgetsearch-profile

## Related Issue/Ticket

https://jira.di2e.net/browse/UOT-109813 (Build AI Inventory Clone)
https://jira.di2e.net/browse/UOT-109854 (Implement Keyword Search JAIC AI Keyword list)
https://jira.di2e.net/browse/UOT-110824 (Build Budget Project/BLI Profile Page Component ("updated" document explorer)

## Testing instructions

- Navigate to http://10.194.9.105:8080/#/budgetsearch, table should eventually load, search should include additional results
- Navigate to http://10.194.9.105:8080/#/budgetsearch-profile, should see the hardcoded profile page screen
- 404 means I need to go run the sql script that adds the clone in 105's postgres
